keps/sig-api-machinery/3157-watch-list/README.md

```diff
@@ -385,13 +385,39 @@ Since the cacheWatcher starts processing the cacheWatcher.incoming channel only
 In that case, it will be added to the list of blockedWatchers and will be given another chance to deliver an event after all nonblocking watchers have sent the event.
 All watchers that have failed to deliver the event will be closed.
 
+Closing the watchers would make the clients retry the requests and download the entire dataset again even though they might have received a complete list before.
+
+For an alpha version, we might simply delay closing the watch request until all data is sent to the client.
```

Comment:
we will delay

Reply:
We expect this to behave well even in heavily loaded clusters. To increase confidence in the approach, we will collect metrics ...